tools/wasi/wasi.hpp

```diff
@@ -13,6 +13,9 @@ struct Instance;
 
 namespace wasi
 {
+class UVWASI;
+extern UVWASI* uvwasi;
```

Comment:
Is this needed for mocking?

Reply:
Yes.

Alternatively maybe It could be declared in `wasi_test.cpp` and another time in `wasi.cpp`